docs/pages/admin-guides/access-controls/sso/sso.mdx

```diff
@@ -411,6 +411,106 @@ values to match your identity provider:
 
 At this time, the `spec.provider` field should not be set for any other identity providers.
 
+## Configuring SSO for MFA checks
+
+Teleport administrators can configure Teleport to delegate MFA checks to an 
+SSO provider as an alternative to registering MFA devices directly with the Teleport Cluster.
+This allows Teleport users to use MFA devices and custom flows configured in the SSO provider
+to carry out privileged actions in Teleport, such as:
+
+- MFA device management (e.g. register or delete a webauthn device in Teleport)
+- [Per-session MFA](../guides/per-session-mfa.mdx)
+- [Moderated sessions](../guides/moderated-sessions.mdx)
+- [Admin actions](../guides/mfa-for-admin-actions.mdx)
+
+<Admonition type="warning">
+  SSO MFA is an enterprise feature. Only OIDC and SAML auth connectors are supported.
+</Admonition>
+
+### Considerations
+
+Administrators may want to consider enabling this feature for the following benefits:
+
+- All authentication (login and MFA) goes through the IDP, reducing administrative overhead
+- Make custom MFA flows, such as prompting for 2 distinct devices for a single MFA check
+- Integrate with non-webauthn devices supported directly by your IDP
+- Enable new SSO users to carry out privileged actions without requiring them to register their device through Teleport first
```

Comment:
I think we could condense this section with the one before by removing the "Considerations" heading" and one of the bullets. Then readers can get into the heart of the section a little more quickly.

Reply:
Removed the last bullet point and move it up as suggested.